tests/test_cli.py

```diff
@@ -9,7 +9,7 @@
 import pytest
 from click.testing import CliRunner
 
-from rpmlb.cli import run
+from rpmlb import cli
```

Comment:
This line is to use 2 methods in rpmlb.cli module in `test_cli.py` .

The way is from
http://docs.python-guide.org/en/latest/writing/structure/
> Best
> x = modu.sqrt(4)  # sqrt is visibly part of modu's namespace